src/components/ListOrgs.vue

```diff
@@ -431,10 +476,51 @@ const showCode = async (selectedOrg) => {
   }
 };
 
+const onEditButtonClick = (event) => {
+  isEditModalEnabled.value = true;
+  currentEditOrgId.value = _get(event, 'id', null);
+};
+
+const closeEditModal = () => {
+  isEditModalEnabled.value = false;
+  currentEditOrgId.value = null;
+};
+
 const closeDialog = () => {
   isDialogVisible.value = false;
 };
 
+const updateOrgData = async () => {
+  isSubmitting.value = true;
+  await roarfirekit.value
+    .createOrg(
+      activeOrgType.value,
+      localOrgData.value,
+      _get(localOrgData.value, 'testData', false),
+      _get(localOrgData.value, 'demoData', false),
+      currentEditOrgId.value,
+    )
+    .then(() => {
+      isSubmitting.value = false;
+      closeEditModal();
+      toast.add({
+        severity: TOAST_SEVERITIES.SUCCESS,
+        summary: 'Updated',
+        detail: 'Organization data updated successfully!',
+        life: TOAST_DEFAULT_LIFE_DURATION,
+      });
+    })
+    .catch((error) => {
+      toast.add({
+        severity: TOAST_SEVERITIES.ERROR,
+        summary: 'Unexpexcted error',
+        detail: `Unexpected error occurred: ${error.message}`,
+        life: TOAST_DEFAULT_LIFE_DURATION,
+      });
+      isSubmitting.value = false;
+    });
```

Comment:
This is swallowing the error, or better preventing the error from being logged/reported to Sentry as you're catching the error and then only displaying it to the user. 

Instead, it would be better to report the error back to Sentry as it was done here by @Emily-ejag: 
https://github.com/yeatmanlab/roar-dashboard/blob/65953f10e6aa7189e9c145008616d763bd2a9cb6/src/components/ListOrgs.vue#L350-L358